src/parse.cpp

```diff
@@ -1771,6 +1772,36 @@ expression* parser::parse_object_literal() {
         break;
       }
 
+      case token_type::equal: {
```

Comment:
Blocking: This code looks like a copy-paste from line 1564. This is a lot of duplicated code. Let's try to avoid the copy-paste.